fastly/resource_fastly_tls_subscription.go

```diff
@@ -25,6 +25,7 @@ func resourceFastlyTLSSubscription() *schema.Resource {
 			customdiff.ForceNewIf("configuration_id", resourceFastlyTLSSubscriptionStateNotIssued),
 			customdiff.ForceNewIf("domains", resourceFastlyTLSSubscriptionStateNotIssued),
 			customdiff.ForceNewIf("common_name", resourceFastlyTLSSubscriptionStateNotIssued),
+			resourceFastlyTLSSubscriptionSetNewComputed,
```

Comment:
It might be marginally more readable to use the `customdiff.ComputedIf()` helper method instead of calling `d.SetNewComputed()` directly? I did a similar thing in #388 but up to you